### PR TITLE
Fixed a sentence

### PR DESCRIPTION
**Pull request by deepug9787**

### CODE REVIEW

File: files/en-us/web/css/css_grid_layout/relationship_of_grid_layout/index.md [hunk 1]
```diff
@@ -318,7 +318,8 @@ This isn't quite the same as flexbox. In the flexbox example, the items are larg
 
 {{ EmbedLiveSample('A_flexible_number_of_tracks', '500', '170') }}
 
-We now have the ability to create a grid with a flexible number of flexible tracks, but see items laid out on the grid aligned by rows and columns at the same time.
+So not only do we now have the ability to create a grid with a dynamic number of flexible tracks, we can also have the items laid out
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/css_grid_layout/relationship_of_grid_layout/index.md:321]: The original was written when this was a new feature. I suggest we get rid of the "now" new language.

**Review by estelle — CHANGES_REQUESTED** [review]

**Comment by joziasmartini** [comment]
Hey @estelle, remove this trailing space after the sentence to unblock the merge.

image

**Comment by teoli2003** [comment]
@deepug9787 You should fix the sentence as Estelle requested, too.

**Comment by deepug9787** [comment]
@teoli2003 If you notice, that sentence is actually a conclusion of a section titled "A flexible number of tracks", which explains how we can achieve a flexbox-like wrapping functionality using the `auto-fit` and` minmax()` function in grid. So the "now" in that sentence refers to the result of the steps we've taken in that section to achieve the desired functionality, and not the current state of browser support for those grid features. 

Could you please confirm whether you want me to fix the sentence as @estelle requested?

File: files/en-us/web/css/css_grid_layout/relationship_of_grid_layout/index.md [hunk 2]
```diff
@@ -318,7 +318,7 @@ This isn't quite the same as flexbox. In the flexbox example, the items are larg
 
 {{ EmbedLiveSample('A_flexible_number_of_tracks', '500', '170') }}
 
-We now have the ability to create a grid with a flexible number of flexible tracks, but see items laid out on the grid aligned by rows and columns at the same time.
+So not only do we now have the ability to create a grid with a dynamic number of flexible tracks, we can also have the items laid out on the grid aligned by rows and columns.
```
Comment (estelle) [inline files/en-us/web/css/css_grid_layout/relationship_of_grid_layout/index.md:321]:
```suggestion
With grid layout, we can create a grid with a dynamic number of flexible tracks and have the items laid out on the grid aligned by rows and columns.
```

**Review by estelle — APPROVED** [review]

**Pull request merged**